src/envo/envo-edit.owl

```diff
@@ -6392,6 +6396,7 @@ SubClassOf(<http://purl.obolibrary.org/obo/ENVO_00000103> ObjectSomeValuesFrom(<
 
 # Class: <http://purl.obolibrary.org/obo/ENVO_00000104> (undersea feature)
 
+AnnotationAssertion(<http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/ENVO_00000104> "A hydrographic feature which is located below the surface of a marine water body."@en)
```

Comment:
hmmm, wouldn't features on the sea floor (sand waves, biogenic mounds, rift valleys, seamounts, reefs) also be considered undersea features?   these are not hydrographic.